go.mod

```diff
@@ -65,7 +65,7 @@ require (
 	github.com/AzureAD/microsoft-authentication-library-for-go v0.5.1
 	github.com/Masterminds/semver/v3 v3.2.0
 	github.com/Masterminds/sprig/v3 v3.2.3
-	github.com/aws/aws-sdk-go v1.44.300
```

Comment:
Could we separate out the drone vs. AWS SDK bumps? I am okay with approving drone, but do know which team uses the AWS SDK version.

Reply:
For future reference, the `aws-sdk-go` dependency is being brought via 4 other dependencies atm. I am assuming this is mostly related to provisioning:


![image](https://github.com/rancher/rancher/assets/5452977/42ea3d4a-32d6-40ee-b9b5-86342a873d91)